Documentation/HowToUseWCFInContainers.md

```diff
@@ -0,0 +1,142 @@
+using System;
+using System.ServiceModel;
+
+namespace ConsoleApp1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var url = Environment.GetEnvironmentVariable("url");
+            var helloEndpoint = new EndpointAddress(url);
+            var binding = new BasicHttpBinding();
+            var factory = new ChannelFactory<IService1>(binding, helloEndpoint);
+            var client = factory.CreateChannel();
+            var data = client.GetData(5);
+            Console.WriteLine(data);
+        }
+    }
+}
+~~~
+
+Finally, we�ll need a project file. Unfortunately the project file may look different depending on the version of .NET core SDK you�re going to use in the container. One way to generate a project file compatible with a particular version of SDK is to run the following:
+
+~~~
+mkdir wcfClient
+cd wcfClient
+dotnet new console
+~~~
+
+(make sure the directory you�re running this in doesn�t already have a project file) and use the newly created project file as a template for your WCF client project. The last step is to add references to WCF.NET Core project like the one below. Again, depending on the SDK version, the format may change. The one that we�re providing here will be compatible with the latest 1.1.1 nightly build (wcfClient.csproj):
+
+~~~
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETCore.App">
+      <Version>1.0.1</Version>
+    </PackageReference>
+    <PackageReference Include="System.Private.ServiceModel">
+      <Version>4.3.0</Version>
+    </PackageReference>
+    <PackageReference Include="System.ServiceModel.Http">
+      <Version>4.3.0</Version>
+    </PackageReference>
+    <PackageReference Include="System.ServiceModel.NetTcp">
+      <Version>4.3.0</Version>
+    </PackageReference>
+    <PackageReference Include="System.ServiceModel.Primitives">
+      <Version>4.3.0</Version>
+    </PackageReference>
+    <PackageReference Include="System.ServiceModel.Duplex">
+      <Version>4.3.0</Version>
+    </PackageReference>
+    <PackageReference Include="System.ServiceModel.Security">
+      <Version>4.3.0</Version>
+    </PackageReference>
+  </ItemGroup>
+</Project>
+~~~
+
+#### To sum it up: you have a WCF default template (or your own) service running and a folder called �wcfClient� with the client app that has 3 files: wcfClient.csproj, program.cs, and IService1.cs.
+
+### 2. Container base image
+.NET Core Runtime Docker images are available for both released versions and for nightly builds (https://hub.docker.com/r/microsoft/dotnet-nightly/). One can get images that have only the .NET Core runtime or both the runtime and the SDK. We�re going to use Nano Server and Linux SDK images for the latest 1.1.1 nightly build (which are compatible with the project file that we created above).
```

Comment:
One benefit of containers is that you can quickly try latest & greatest version without a need to install anything. In this section you're picking the right container image for your app. The next section shows examples of the dockerfiles that will use the image you chose here. The choice of 1.1.1 image is in line with the spirit of "quickly trying latest and greatest" via Docker images - in this case - the latest update for a shipped product version.